utils/sub8_diagnostics/scripts/thruster_spinner.py

```diff
@@ -31,7 +31,10 @@ def ports_from_layout(layout):
             for name in thruster_names:
                 port_dict[name] = thruster_port
 
-            msg += "\n\tMotor id's on port: {}".format(thruster_port.motor_ids_on_port)
+            msg += "\n\tMotor id's on port: {}{}{}".format(
```

Comment:
How would you do it if msg was a printer type? Tbh, honest this is harder to use than it needs to be because it doesn't have any docstrings.